scripts/install-chef-suse.sh

```diff
@@ -575,9 +584,12 @@ if [ -n "$CROWBAR_FROM_GIT" ]; then
         add_ibs_repo http://dist.suse.de/ibs/Devel:/Cloud:/6/SLE_12_SP1/ cloud
     fi
 
+    # Be sure that systemd-logger is deinstalled
+    zypper --non-interactive remove systemd-logger
```

Comment:
that looks like a workaround for https://github.com/crowbar/crowbar-core/pull/160